config/routes.rb

```diff
@@ -4,19 +4,24 @@
 
   # Sufia generated routes
   blacklight_for :catalog
-  devise_for :users
+
+  scope "/admin" do
+    devise_for :users
+  end
+
   Hydra::BatchEdit.add_routes(self)
-  # This must be the very last route in the file because it has a catch-all route for 404 errors.
-    # This behavior seems to show up only in production mode.
-  mount Sufia::Engine => '/'
+  # This must be the very last route in the file because it has a catch-all
+  # route for 404 errors.
+  # This behavior seems to show up only in production mode.
+  mount Sufia::Engine => "/admin"
 
   resources :searches
```

Comment:
Do we need this anymore? also it might be better to limit the routes here `only: [:index]`